src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/DefendantEntity.java

```diff
@@ -53,9 +53,9 @@
 
     @ToString.Exclude
     @ManyToOne(optional = false)
-    @JoinColumn(name = "COURT_CASE_ID", referencedColumnName = "id", nullable = false)
+    @JoinColumn(name = "FK_HEARING_ID", referencedColumnName = "id", nullable = false)
```

Comment:
Added FK_ prefix to new foreign keys to reduce confusion because we previously had `case_id` and `court_case_id` which isn't very clear